onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -604,10 +607,12 @@ std::unique_ptr<IExecutionProvider> CreateExecutionProviderInstance(
 
       
     }
-    auto p = onnxruntime::CreateExecutionProviderFactory_OpenVINO(&params)->CreateProvider();
-    // Reset global variables config to avoid it being accidentally passed on to the next session
-    openvino_device_type.clear();
-    return p;
+    if (std::shared_ptr<IExecutionProviderFactory> tensorrt_provider_openvino = onnxruntime::CreateExecutionProviderFactory_OpenVINO(&params)) {
+        auto p = tensorrt_provider_openvino->CreateProvider();
+        // Reset global variables config to avoid it being accidentally passed on to the next session
+        openvino_device_type.clear();
+        return p;
+    }
```

Comment:
can we add a similar check/throw exception to what we do for CUDA_PATH for OpenVINO?
the env variable is INTEL_OPENVINO_DIR

Reply:
Okay, will do

Reply:
requirements page is at https://onnxruntime.ai/docs/execution-providers/OpenVINO-ExecutionProvider.html#requirements